### DISCUSSION TITLE
Starting a new project and repository name About_Keploy

### DISCUSSION THREAD

**Discussion by Ayush7614** [keploy/keploy#128] [Ideas]
In this repo we define all the things of Keploy Organization from staring to end
- Contributing to Keploy
- Working Groups
- How to Join Keploy
- About Keploy
- About Community
- Meetings
- Mentorships

**Comment by Ayush7614** [comment]
cc @nehagup @slayerjain

**Reply by nehagup** [reply]
Thanks for taking this initiative @Ayush7614 

It would be awesome if you could please elaborate on what will be the purpose of the repository containing the mentioned contents. 

Also, if you're talking about a template repo that can be used as a base for every new repo - please check this one - https://github.com/keploy/template

**Comment by Ayush7614** [comment]
This Repo Contains all the information about Keploy and it is specially the documents of the Keploy Orgazination Like in contributing to Keploy Section Contains how anyone can contribute to Keploy and many things, in working Groups section all the meeting information was there, in how to join Keploy Section anyone newcomers, users, Contributers how they join Keploy like slack channel and all things, About Keploy Section contain introduction, history and all things about Keploy, About community section contains people's of community, initiative of community and all things, in mentorship section all things about GitHub Externship, gsoc and all things

**Reply by nehagup** [reply]
@Ayush7614 We've created this template https://github.com/keploy/template repo for this purpose. Please feel free to add more changes and raise a PR there. 🙂

**Comment by Ayush7614** [comment]
Cc @nehagup
